mobilitypath_publisher/src/mobilitypath_publisher.cpp

```diff
@@ -131,10 +131,6 @@ namespace mobilitypath_publisher
                 traj.offsets.push_back(offset);
             }
         }
-        //according to msg standard, ecef_location is in meters
```

Comment:
Is the message standard wrong or was this comment incorrect before?

Reply:
the comment and this /100 logic was incorrect before. So I updated it to use cm, so we don't have this code. Rest of the packages also use this metrics